texthero/preprocessing.py

```diff
@@ -17,6 +17,34 @@
 
 from typing import List, Callable
 
+# REGEX pattern constants
+PATTERN_REMOVE_DIGITS_BLOCK = r"\b\d+\b"
+PATTERN_REMOVE_PUNCTUATION = rf"([{string.punctuation}])+"
+PATTERN_STOPWORD_TOKENIZER = r"""(?x)                          # Set flag to allow verbose regexps
+                                \w+(?:-\w+)*                              # Words with optional internal hyphens 
+                                | \s*                                     # Any space
+                                | [][!"#$%&'*+,-./:;<=>?@\\^():_`{|}~]    # Any symbol 
+                                """
+PATTERN_REMOVE_ROUND_BRACKETS = r"\([^()]*\)"
+PATERN_REMOVE_CURLY_BRACKETS = r"\{[^{}]*\}"
+PATTERN_REMOVE_SQUARE_BRACKETS = r"\[[^\[\]]*\]"
+PATTERN_REMOVE_ANGLE_BRACKETS = r"<[^<>]*>"
+PATTERN_REMOVE_HTML_TAG = r"""(?x)                    # Turn on free-spacing
+                            <[^>]+>                             # Remove <html> tags
+                            | &([a-z0-9]+|\#[0-9]{1,6}|\#x[0-9a-f]{1,6}); # Remove &nbsp;
+                            """
+
+
+def GET_PATTERN_TOKENIZATION(punct: str) -> str:
```

Comment:
Global functions are always lowercased
"Returns the standart tokenisation pattern": not particularly meaningful
"Returns" -> "Return"

Reply:
ah thanks 🥇 I was unsure about the python style guide. But now it is a private function